packages/aws-cdk/test/integ/cli/cdk-helpers.ts

```diff
@@ -4,15 +4,15 @@ import * as path from 'path';
 import { cloudFormation, deleteBucket, deleteImageRepository, deleteStacks, emptyBucket, outputFromStack, testEnv } from './aws-helpers';
 import { writeOutput } from './corking';
 
-export const INTEG_TEST_DIR = path.join(os.tmpdir(), 'cdk-integ-test2');
-
 // create a unique stack name prefix for this test test run. this is passed
 // through an environment variable to app.js so that all stacks use this prefix.
 const timestamp = new Date().toISOString().replace(/[^0-9]/g, '');
 export const STACK_NAME_PREFIX = `cdktest-${timestamp}`;
+export const INTEG_TEST_DIR = path.join(os.tmpdir(), `cdk-integ-${timestamp}`);
```

Comment:
There's one downside which is that these tempdirs are never cleaned up. So there ya go :)

Reply:
true true